modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/macros/Workflow.macro

```diff
@@ -407,8 +407,12 @@ definition {
 
 		Click(locator1 = "Icon#SIDEBAR_HEADER_ACTION_VERTICAL_ELLIPSIS_BUTTON");
 
+		Pause(value1 = 5000);
+
 		MenuItem.click(menuItem = "Edit");
 
+		Pause(value1 = 5000);
```

Comment:
Hey @seltong,

use `WaitForElementPresent` or something like this:

```
while ((IsElementNotPresent(locator1 = "WorkflowAllItems#WORKFLOW_ALL_ITEMS_FILTER_BY_ASSIGNEE_VALUE")) && (maxIterations = "10")) {
				Refresh();

				WaitForSPARefresh();

				Click(locator1 = "WorkflowAllItems#WORKFLOW_ALL_ITEMS_FILTER_BY");
			}
```

we've talked to @Luizcarlosqueiroz and he said the `Pause` function is no longer used and we should avoid